blueprints/jsonapi-resource/index.js

```diff
@@ -57,19 +57,16 @@ module.exports = {
     var modelOptions = merge({}, options, {
       entity: {
         name: entityName ? inflection.singularize(entityName) : ''
-      },
-      originBlueprintName: 'jsonapi-model'
+      }
     });
 
     var otherOptions = merge({}, options);
 
     return Promise.all([
       this._processBlueprint(type, 'jsonapi-model', modelOptions),
-      this._processBlueprint(type, 'addon-import', modelOptions),
```

Comment:
I'm sure it's necessary, but not when generating a `jsonapi-resource`, right?

I'll give this some more attention/testing with your jr-test repo.